graphite/graphite_tcp.go

```diff
@@ -129,6 +129,7 @@ func (t *TCPServer) handleConnection(conn net.Conn) {
 		// Read up to the next newline.
 		buf, err := reader.ReadBytes('\n')
 		if err != nil {
+			flush <- struct{}{}
```

Comment:
OK, I can believe that. I wanted to try out both ways, but I can see how it looks raw.